src/warplib/gatherWarplibImports.ts

```diff
@@ -4,14 +4,11 @@ import { parseMultipleRawCairoFunctions } from '../utils/cairoParsing';
 import { glob } from 'glob';
 
 export const warplibImportInfo = glob
-  .sync('warplib/**/*.cairo')
+  .sync('warplib/src/**/*.cairo')
   .reduce((warplibMap, pathToFile) => {
     const rawCairoCode = fs.readFileSync(pathToFile, { encoding: 'utf8' });
 
-    const importPath = pathToFile
-      .split('/')
-      .join('.')
-      .slice(0, pathToFile.length - '.cairo'.length);
+    const importPath = 'warplib/' + pathToFile.slice('warplib/src/'.length, -'.cairo'.length);
```

Comment:
What has changed that we don't use `.` as separator here anymore?

Reply:
I change the separator to /

Reply:
Why is the `src` part not needed in the import path?

Reply:
@cicr99 It's because it's not the system path, it's import path separated by slashes

Reply:
And since it's an import path it would be better to write it that way:
```suggestion
    const importPath = encodePath(['warplib', pathToFile.slice('warplib/src/'.length, -'.cairo'.length).split(path.sep)]);
```

Though paths mapping/encoding functionality will require a refactor in future

Reply:
I agree. I would like to postpone the addition of  encodePath to a future PR because it is not that straightforward, due to circular dependency between importFuncGenerator and gatherWarplibImports. Moving the `encodePath` function to gatherWarplib would solve the issue, but I think it is alright where it is located.

I think the best solution is a small movement of files where we joined all three major files: imporFuncGenerator, importPaths and gatherWarplibImports, to their own import folder, inside utils.

Tried doing it inside of this PR but there are lot of updating to do across the whole project